frontend/app/src/components/StatusWidget/StatusWidget.tsx

```diff
@@ -247,6 +247,14 @@ const StatusWidget: React.FC<StatusWidgetProps> = ({
         }
       case ConnectionState.CONNECTED:
         return undefined
+      case ConnectionState.STATIC_CONNECTING:
+        return {
+          icon: Ellipses,
+          label: "Connecting",
+          tooltip: "Connecting to Notebooks",
```

Comment:
Not fully close to the project, but are we pigeon holing ourselves to the conversation of Notebooks? Just weird to have a general concept (STATIC_CONNECTING), and then specify it's connecting to Notebooks.

If the Notebook specification is important, I would call it STATIC_NOTEBOOK_CONNECTING or something.

I lean towards keeping it generic because we may want to generalize this as a feature, right?